alchemy/test/cloudflare/do-state-store.test.ts

```diff
@@ -0,0 +1,251 @@
+import { describe, expect } from "bun:test";
+import { alchemy } from "../../src/alchemy.js";
+import { DOFSStateStore } from "../../src/cloudflare/do-state-store/index.js";
+import { destroy } from "../../src/destroy.js";
+import { BRANCH_PREFIX } from "../util.js";
+
+// must import this or else alchemy.test won't exist
+import "../../src/test/bun.js";
+
+const test = alchemy.test(import.meta, {
+  prefix: BRANCH_PREFIX,
+});
```

Comment:
A great way to test this would be to use it.

```ts
const test = alchemy.test(import.meta, {
  prefix: BRANCH_PREFIX,
  stateStore: (scope) => new DOFSStateStore(scope, {
      autoDeploy: false,
      url: "https://test-worker.example.workers.dev",
      workerName: `${BRANCH_PREFIX}-test-worker`,
      basePath: "/test-alchemy",
    })
});
```